### ISSUE TITLE
Troubles with make

### ISSUE THREAD

**Issue by boabjohn** [microsoftarchive/redis#498]
Hi guys, I'm on a Win7 64-bit machine and all I need is redis-cli so that I can run a clear cache command on a remote server (Pantheon).

Not sure what the easiest way to achieve this is (aside from going fully linux!) but I have cygwin running and am trying my best.
1. Download the .zip and unpack at c:/redis/redis-3.0
2. Navigate here with cygwin and run make
3. The attached snippet shows make trying to run and then encountering a bunch of errors around here:

make[3]: Entering directory '/cygdrive/c/redis/redis-3.0/deps/hiredis'
cc -std=c99 -pedantic -c -O3 -fPIC  -Wall -W -Wstrict-prototypes -Wwrite-strings -g -ggdb  net.c
net.c:1:0: warning: -fPIC ignored for target (all code is position independent)
 /\* Extracted from anet.c to work properly with Hiredis error reporting.

[redis-make.log.txt](https://github.com/MSOpenTech/redis/files/431739/redis-make.log.txt)

Any pointers much appreciated!

**Comment by chester89** [comment]
@boabjohn I think it's much easier just to use redis-cli.exe that ships with Redis

**Comment by loveheartfarm** [comment]
Thanks @chester89, and apologies for being clueless, but redis-cli.exe is precisely what I'm looking for and don't understand how to get.

The download from this repo does not contain any *.exe files: those have to be compiled, right?

So that's what I've outlined above...my failure to make the compiled redis.

After searching again I've found this: http://codingsteps.com/install-redis-on-windows/
Will give it a go. Thanks.
